lib/mix/tasks/rustler.download.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Mix.Tasks.Rustler.Download do
+  @shortdoc "Download precompiled NIFs and build checksums"
+
+  @moduledoc """
+  This is responsible for downloading the precompiled NIFs for a given module.
+
+  It will also save the checksum file in the proper location in case it is not
+  present. This is important because we need to calculate the checksum before
+  extracting the NIF to the proper location.
+  """
+
+  use Mix.Task
+
+  alias Html5ever.Precompiled
+
+  @impl true
+  def run([module_name | maybe_flags]) do
+    urls =
+      case maybe_flags do
+        ["--all"] ->
+          Precompiled.available_nif_urls(module_name)
+
+        ["--only-target"] ->
```

Comment:
Did you name it --only-target because for Nerves this is the target configuration? Maybe we should call it `--only-local` then? Or `--only-current`?